### DISCUSSION TITLE
Next13  Input and hydration

### DISCUSSION THREAD

**Discussion by darkjeff99** [mantinedev/mantine#3762] [Q&A] [answered]
Hi all.

I'm trying to use Nextjs 13 whit app dir and Mantine. Everything is fine until I use an input. When I add a label or a placeholder I  have an hydration issue. 

<img width="909" alt="image" src="https://user-images.githubusercontent.com/1247054/224574089-c7ea8399-f606-4951-9457-5997fafb6cb4.png">

It is an new project with 
``` "@emotion/react": "^11.10.6",
    "@mantine/core": "^6.0.1",
    "@mantine/hooks": "^6.0.1",
    "@mantine/modals": "^6.0.1",
    "@tabler/icons-react": "^2.10.0",
    "@types/node": "18.14.6",
    "@types/react": "18.0.28",
    "@types/react-dom": "18.0.11",
    "eslint": "8.35.0",
    "eslint-config-next": "13.2.3",
    "next": "^13.2.4",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-hook-form": "^7.43.5",
    "typescript": "4.9.5"
```

Do we have something else to do ?

**Comment by rtivital** [comment, marked as answer]
You need to setup Mantine to work with app dir – https://github.com/mantinedev/mantine-next-template/tree/next-13-app
Note that it is experimental feature and we do not provide any official documentation on it yet. You can follow this issue to learn more –  https://github.com/mantinedev/mantine/issues/2815
